modules/bullet/godot_collision_dispatcher.h

```diff
@@ -31,14 +31,8 @@
 #ifndef GODOT_COLLISION_DISPATCHER_H
 #define GODOT_COLLISION_DISPATCHER_H
 
-#include <cstdint>
```

Comment:
This and a few other cleanups in the bullet module were tested on `3.x` since we don't build bullet on `master` right now.